Server/GameEngine.cpp

```diff
@@ -60,25 +61,26 @@ vec3 GameEngine::movementInputToVector(int movementInput) {
 	return glm::normalize(movement);
 }
 
-void GameEngine::movePlayers(vector<PlayerInputs> playerInputs) {
+void GameEngine::movePlayers(vector<PlayerInputs> & playerInputs) {
 	vector<int> aggregatePlayerMovements;
-
+	//cout << "playerInputs size: " << playerInputs.size() << endl;
 	// Use bitwise or to get all player inputs within one server tick
-	for (int i = 0; i < NUM_PLAYERS; i++) {
+	for (int i = 0; i < gameState.players.size(); i++) {
```

Comment:
We can replace the loop by using the [vector constructor that sets the initial size](http://www.cplusplus.com/reference/vector/vector/vector/) instead when creating `aggregatePlayerMovements`